src/rocksdb.rs

```diff
@@ -468,31 +421,15 @@ impl DB {
         self.write_opt(batch, &wo)
     }
 
-    pub fn get_opt(&self,
-                   key: &[u8],
-                   readopts: &ReadOptions)
-                   -> Result<Option<DBVector>, Error> {
+    pub fn get_opt(&self, key: &[u8], readopts: &ReadOptions) -> Result<Option<DBVector>, Error> {
         if readopts.inner.is_null() {
-            return Err(Error::new("Unable to create rocksdb read options.  \
-                                   This is a fairly trivial call, and its \
-                                   failure may be indicative of a \
-                                   mis-compiled or mis-loaded rocksdb \
-                                   library."
-                .to_string()));
+            return Err(Error::new("Unable to create rocksdb read options. This is a fairly trivial call, and its failure may be indicative of a mis-compiled or mis-loaded rocksdb library.".to_owned()));
```

Comment:
I see what you mean here... Personally, I have this setting because I hate hard line wraps (it's inflexible with regards to editor size, it's hard to maintain when editing/renaming things, and really looks odd for heavily indented code). So I do try to let the editor do the (soft) wrapping wherever possible, since that's it's job as I see it... Nonetheless, I know there are plenty of folk out there that think otherwise, so do what you like, and sorry if it's an inconvenience. :-)